### DISCUSSION TITLE
cannot find css styles

### DISCUSSION THREAD

**Discussion by simon-muyezhang** [afconsult/quill-mention#417] [Q&A] [answered]
![1652064487(1)](https://user-images.githubusercontent.com/10706610/167332190-5efcacd1-50f3-434d-bec0-aea9596f98c3.png)

**Comment by amitpatil321** [comment]
Found any solution ? I am also having same issue :/

**Comment by akshay-aravind** [comment]
Why is this issue closed if it still exist?
Do anyone have a solution for this?

**Comment by csculley** [comment, marked as answer]
If your build system seems to not be automatically importing styles, then you may be able to manually import it via `import "quill-mention/dist/quill.mention.min.css";` in js or `@import "quill-mention/dist/quill.mention.min.css";` in css. However, this is usually indicative of a problem with the build system configuration and out of scope of an issue in the project. I'll convert this into a discussion for now!

**Comment by ngocken95** [comment]
Why is this issue closed if it still exist?
Do anyone have a solution for this?
